README.md

```diff
@@ -377,14 +377,14 @@ You can redirect the log output to a logger of your own by specifying a custom l
 ```java
 import io.ably.lib.util.Log.LogHandler;
 
-ClientOptions opts = new ClientOptions(key);
-opts.logHandler = new LogHandler() {
-	public void println(int severity, String tag, String msg, Throwable tr) {
-		/* handle log output here ... */
-	}
-};
-AblyRest ably = new AblyRest(opts);
-...
+ClientOptions opts=new ClientOptions(key);
+  opts.logHandler=new LogHandler(){
+public void println(int severity,String tag,String msg,Throwable tr){
+  /* handle log output here ... */
+  }
+  };
+  AblyRest ably=new AblyRest(opts);
+  ...
```

Comment:
This seems out of scope of this PR (and the formatting applied is a bit odd :sweat_smile:), please remove it :wink:

Reply:
done